### ISSUE TITLE
question

### ISSUE THREAD

**Issue by IvyFeng413** [2020-GPW-L5K2-C/CD-Bus-Company#41]
hi Jons,
In the paper and code submission, do we need each member to upload or only need one member to upload？
@jslemmer

**Comment by 9sa6c** [comment]
No, only 1 person of your group needs to upload all the required elements
